src/vm/loaderallocator.hpp

```diff
@@ -485,7 +485,6 @@ typedef VPTR(LoaderAllocator) PTR_LoaderAllocator;
 class GlobalLoaderAllocator : public LoaderAllocator
 {
     VPTR_VTABLE_CLASS(GlobalLoaderAllocator, LoaderAllocator)
-    VPTR_UNIQUE_OVERRIDE(VPTRU_LoaderAllocator+1)
```

Comment:
In the last build, it looked like this method worked, but it complained about VPtrSize not being marked with override. That might be easy enough to fix at line 1821 before this commit. Otherwise, it feels a bit scary to hide base virtual members and not override them. It would probably be better to suppress the warning using pragma for this whole file, and if this continues to be an issue, it could be turned off globally in src/pal/tools/clang-compiler-override.txt.